src/sentry/snuba/spans_indexed.py

```diff
@@ -49,3 +56,57 @@ def query(
 
     result = builder.process_results(builder.run_query(referrer))
     return result
+
+
+def timeseries_query(
```

Comment:
This will need to include `tenant_ids` in the query otherwise snuba will reject it

https://vanguard.getsentry.net/p/clgxyod9m0000s60lazmrg8j3